{"version":"3.8.0","messages":["\nAn error occurred while loading ./spec/operations_spec.rb.\n\u001b[31mFailure/Error:\u001b[0m\n\u001b[31m  \u001b[0mrequire_relative ./spec_helper.\u001b[1;30m#!/usr/bin/env ruby -wKU\u001b[0m\u001b[0m\n\u001b[31m  \u001b[0m\n\u001b[31m  \u001b[0mdescribe \u001b[31m\u001b[1;31m\"\u001b[0m\u001b[31mOperations\u001b[1;31m\"\u001b[0m\u001b[31m\u001b[0m \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m    \u001b[0m\u001b[32mlet\u001b[0m(\u001b[33m:file_contents\u001b[0m) { \u001b[1;34;4mFile\u001b[0m.read(\u001b[1;34;4mFile\u001b[0m.join(\u001b[1;34;4mFile\u001b[0m.dirname(\u001b[1;36m__FILE__\u001b[0m), \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31m..\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m, \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mlib/operations.rb\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m)) }\u001b[0m\n\u001b[31m  \u001b[0m\n\u001b[31m    \u001b[0m\u001b[32mdescribe\u001b[0m \u001b[31m\u001b[1;31m\"\u001b[0m\u001b[31m#unsafe?\u001b[1;31m\"\u001b[0m\u001b[31m\u001b[0m \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m      \u001b[0m\u001b[32mit\u001b[0m \u001b[31m\u001b[1;31m'\u001b[0m\u001b[31mreturns true if speed is greater than 60\u001b[1;31m'\u001b[0m\u001b[31m\u001b[0m \u001b[32mdo\u001b[0m\u001b[0m\n\u001b[31m        \u001b[0manswer = unsafe?(\u001b[1;34m79\u001b[0m)\u001b[0m\n\u001b[31m        \u001b[0m\u001b[32mexpect\u001b[0m(answer).to eq(\u001b[1;36mtrue\u001b[0m)\u001b[0m\n\u001b[31m      \u001b[0m\u001b[32mend\u001b[0m\u001b[0m\n\u001b[31m\u001b[0m\n\u001b[31mArgumentError:\u001b[0m\n\u001b[31m  wrong number of arguments (given 0, expected 1)\u001b[0m\n\u001b[36m# ./spec/operations_spec.rb:1:in `require_relative'\u001b[0m\n\u001b[36m# ./spec/operations_spec.rb:1:in `<top (required)>'\u001b[0m\n","No examples found."],"examples":[],"summary":{"duration":0.000559014,"example_count":0,"failure_count":0,"pending_count":0,"errors_outside_of_examples_count":2},"summary_line":"0 examples, 0 failures, 2 errors occurred outside of examples"}